### DISCUSSION TITLE
Extruded SVG image in AR.js throws getUniforms of undefined error.

### DISCUSSION THREAD

**Discussion by ZHAWluci** [AR-js-org/AR.js#387] [Help]
**What is the current behavior?**
I want to import a SVG file into my AR.js scene to be displayed on a marker.
For that I am using the official Three.js SVGLloader[1]. While using this with pure Three.js I am able to load a SVG into my scene and display it. But when I use the exact same function combined with AR.js I get this error:

```
TypeError: Cannot read property 'getUniforms' of undefined
setProgram
https://localhost:3000/libs/three/example/vendor/three.js/build/three.js:24307:26
WebGLRenderer.renderBufferDirect
https://localhost:3000/libs/three/example/vendor/three.js/build/three.js:23313:18
renderObject
https://localhost:3000/libs/three/example/vendor/three.js/build/three.js:24041:11
renderObjects
https://localhost:3000/libs/three/example/vendor/three.js/build/three.js:24011:6
WebGLRenderer.render
https://localhost:3000/libs/three/example/vendor/three.js/build/three.js:23798:33
```

**Please mention other relevant information such as the browser version, Operating System and Device Name**
Windows - Chrome

**What is the expected behavior?**
To  see my extruded SVG logo in AR.js like I see it on Three.js

[1] https://threejs.org/docs/#examples/en/loaders/SVGLoader

**Comment by kalwalt** [comment]
Hi @ZHAWluci have you solved this? Maybe you are using a SVGLoader with a different version of Three.js.
I also moving this issue to discussions because not strictly related to AR.js.
